docs/many-to-many-relations.md

```diff
@@ -293,7 +293,7 @@ Additionally you will have to add a relationship like the following to `Question
 ```typescript
 // category.ts
 ...
-@OneToMany(() => questionToCategory, questionToCategory => questionToCategory.category)
+@OneToMany(() => QuestionToCategory, questionToCategory => questionToCategory.category)
 public questionToCategories: QuestionToCategory[];
```

Comment:
I guess this comment might be a bit old, but shouldn't documentation use the `Relation` type as suggested in the [documentation](https://typeorm.io/#relations-in-esm-projects)?
`Relation<QuestionToCategory[]`

Reply:
I'm not the person addressed, but regardless I'm not sure the `Relation` type wrapper is not needed in cases of arrays (OneToMany, ManyToMany).
I've looked at all instances where it's used, and it's only used with `@OneToOne`.
`clear; grep -rn5 'Relation<'`

https://github.com/typeorm/typeorm/blob/master/src/common/RelationType.ts#L14

The type is a hack, to make typescript not include field type metadata, such as instance of which class this field should contain. Typescript uses `reflect-metadata` and injects references of some classes into the metadata of different classes. This is definitely not needed here because `[];` automatically erases that reference to a `QuestionToCategory` class, and the injected type reference will most likely point to the `Array` class or nothing at all.

Reply:
I didn't know all that, thanks for the info @nikelborm